### ISSUE TITLE
Rcon spams ping command

### ISSUE THREAD

**Issue by xyndex-tech** [MrGraversen/minecraft-rcon#24]
![Screenshot_1](https://github.com/user-attachments/assets/33532dbc-07d9-437b-87d9-28df2c7ef66a)

**Comment by MrGraversen** [comment]
Hello @x3xgit! As part of the connection quality monitoring, the ping command is issued to determine if the RCON/TCP connection is healthy. By default, this occurs once per second by default and not several times a second as your screenshot suggests).

It is possible to tweak this interval when setting up the `ConnectOptions` class. Would you mind sharing some sample code to see if the the setup does anything to change this?

Either way, the `ping` command is harmless and does not incur significant load. In fact, it doesn't even exist in the default Minecraft RCON protocol, and is literally only used to monitor the health of the RCON connection. 😸

**Comment by xyndex-tech** [comment]
> Hello @x3xgit! As part of the connection quality monitoring, the ping command is issued to determine if the RCON/TCP connection is healthy. By default, this occurs once per second by default and not several times a second as your screenshot suggests).
> 
> It is possible to tweak this interval when setting up the `ConnectOptions` class. Would you mind sharing some sample code to see if the the setup does anything to change this?
> 
> Either way, the `ping` command is harmless and does not incur significant load. In fact, it doesn't even exist in the default Minecraft RCON protocol, and is literally only used to monitor the health of the RCON connection. 😸

Yeah thank you, I checked your code before creating this issue and found out this problem but could not except your rcon doing that, so the problem was is Essentials plugin was getting ping command and this plugin was responding back so I deleted command in plugin.yml and it worked nice. Thank you.

**Comment by MrGraversen** [comment]
I see, thank your for the additional details. It may be worth considering adding customization to the connection quality monitoring mechanism, to ensure that conflicts with plugins are possible to circumvent.

**Comment by 3944Realms** [comment]
<img width="2820" height="1707" alt="Image" src="https://github.com/user-attachments/assets/46908f3b-e5eb-4290-8d40-3f6e2548850e" />
i have same trouble. 😓
